src/ucp/core/ucp_mm.c

```diff
@@ -686,24 +686,29 @@ void ucp_mem_print_info(const char *mem_size, ucp_context_h context, FILE *strea
     fprintf(stream, "#\n");
 
     ucs_memunits_to_str(memh->length, memunits_str, sizeof(memunits_str));
-    fprintf(stream, "#  allocated %s at address %p with: ", memunits_str, memh->address);
+    fprintf(stream, "#  allocated %s at address %p with ", memunits_str,
+            memh->address);
 
     if (memh->alloc_md == NULL) {
-        fprintf(stream, "%s ", uct_alloc_method_names[memh->alloc_method]);
+        fprintf(stream, "%s", uct_alloc_method_names[memh->alloc_method]);
     } else {
         for (md_index = 0; md_index < context->num_mds; ++md_index) {
             if (memh->alloc_md == context->tl_mds[md_index].md) {
-                fprintf(stream, "%s ",context->tl_mds[md_index].rsc.md_name);
-
-                uct_memh = ucp_memh2uct(memh, md_index);
-                if ((uct_memh != NULL) && (uct_md_is_hugetlb(context->tl_mds[md_index].md, uct_memh))) {
-                        fprintf(stream, "hugetlb on");
-                }
+                fprintf(stream, "%s", context->tl_mds[md_index].rsc.md_name);
```

Comment:
sorry, I meant `"%s "`

Reply:
line 706 starts with ", " so no need..

Reply:
sliha, I thought we can have several names in a row. Now I see the break